extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -187,14 +197,16 @@ export class PostgresParametersPage extends DashboardPage {
 		this.discardButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
 			label: loc.discardText,
 			iconPath: IconPathHelper.discard,
-			enabled: false
+			enabled: true //TODO
 		}).component();
 
 		this.disposables.push(
 			this.discardButton.onDidClick(async () => {
 				this.discardButton!.enabled = false;
 				try {
 					// TODO
+					this.parametersTable.data = [
```

Comment:
Would suggest not merging this in - just leave it locally for your own dev/testing